configure.ac

```diff
@@ -511,7 +511,7 @@ AC_CHECK_FUNCS([mkdir munmap])
 AC_CHECK_FUNCS([nl_langinfo putenv rmdir])
 AC_CHECK_FUNCS([select setenv socket])
 AC_CHECK_FUNCS([tzset uname])
-AC_CHECK_FUNC([tzset],[:],[AC_DEFINE([_NO_TZSET],1,[Define if not using tzset.])])
+#AC_CHECK_FUNC([tzset],[:],[AC_DEFINE([_NO_TZSET],1,[Define if not using tzset.])])
```

Comment:
Why comment this out instead of deleting it?

Reply:
I guess fear? I figured I would wait for someone to tell me to.

Reply:
I've edited our autotools stuff before and let me tell you, stuff like this makes it harder to work with, as I don't know if that's something possibly useful that I could use as an example, or just cruft from 2008. Please just remove it, let the git log be the record, not comments; we're enlightened.

Reply:
Done